app/pubspec.yaml

```diff
@@ -34,13 +34,13 @@ dependencies:
   yaml: '^2.1.12'
   # pana version to be pinned
   # keep in-sync with app/lib/shared/versions.dart
-  pana: '0.11.8'
+  pana: '0.11.9'
```

Comment:
Pana bump is intentional?

Reply:
Yes, it is a must, otherwise versions don't resolve.